roles/cfg_openwrt/templates/common/config/network.j2

```diff
@@ -33,9 +33,9 @@ config interface '{{ name }}'
     {% if port_needed %}
 	option device '{{ ('br-' + name) if bridge_needed else port }}'
     {% endif %}
+	option macaddr '22:{{ lookup('password', '/dev/null chars="digits" length=2') | lower }}:{{ lookup('password', '/dev/null chars="digits" length=2') | lower }}:{{ lookup('password', '/dev/null chars="digits" length=2') | lower }}:{{ lookup('password', '/dev/null chars="digits" length=2') | lower }}:{{ lookup('password', '/dev/null chars="digits" length=2') | lower }}'
     {% if network.get('enforce_client_isolation') and role == 'corerouter' and
           not bridge_needed %}
-	option macaddr '02:00:00:00:00:01'
```

Comment:
Isn't that breaking client isolation?

Reply:
Removing this line wasn't intentional. However...

This line also sets `option macaddr` , which conflicts with the randomization. 

Two things:
 1) I don't understand how setting a macaddr enforces client isolation
 2) This set below snippet evaluates to true
 ```
   {% set name = network['name'] if 'name' in network else network['role'] %}
   {% set bridge_needed = name in wifi_networks or network.get('mesh_ap') == inventory_hostname or (role == 'corerouter' and 'tunnel_wan_ip' in network) %}
   {% if network.get('enforce_client_isolation') and role == 'corerouter' and  not bridge_needed %}
  option macaddr '02:00:00:00:00:01'
   {% endif %}
```

1) I'm sure there's an answer to. Either way we can maintain this logic.

2) Is kind of convoluted logic. Relying on having macaddr randomization, which may or may not be enabled due to very convoluted logic is sure to create bugs for us down the line.

Because of 2) maybe we should drop this idea and force radios that bridge all VLANs to live behind a managed switch (as they probably should be doing anyway).